LutronLuxmeterIntegration/src/widget.ts

```diff
@@ -1,6 +1,6 @@
 import { CamOverlayDrawingOptions } from 'camstreamerlib/CamOverlayDrawingAPI';
 import { Painter, PainterOptions, Frame } from 'camstreamerlib/CamOverlayPainter/Painter';
-import type { TResult } from './reader';
+import type { TResult } from './LuxMeterReader';
```

Comment:
Super, jeste bych zmenil pro konzistentnost, ze soubory exportujici tridu budou vsechny s velkym pismenem..FE kluci to tak delaj taky..

Reply:
Uznávám, že na Windows nejsou velká písmena v názvech souborů taková zvrácenost.